h2o-core/src/main/java/hex/ModelMetricsMultinomial.java

```diff
@@ -17,7 +17,7 @@
   public final float[] _hit_ratios;         // Hit ratios
   public final ConfusionMatrix _cm;
   public final double _logloss;
-  public final double _mean_per_class_error;
```

Comment:
?

Reply:
Yes actually. In the two cases the final was removed, the value might be overridden in the `ModelMetricsBinomialGeneric` constructor or `ModelMetricsMultinomialGeneric` constructor respectively.